src/Driver/AbstractMySQLDriver.php

```diff
@@ -46,7 +47,14 @@ public function getDatabasePlatform(ServerVersionProvider $versionProvider): Abs
             return new MariaDBPlatform();
         }
 
-        if (version_compare($version, '8.0.0', '>=')) {
+        if (version_compare($version, '8.0.13', '>=')) {
+            return new MySQL8013Platform();
+        }
+
+        if (
+            version_compare($version, '8.0.0', '>=')
+            && version_compare($version, '8.0.13', '<')
```

Comment:
This check is redundant because we've made sure already that we're below 8.0.13 in the `if` block before this one.

Reply:
done